src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -123,58 +121,59 @@ public void setPIDF(double p, double i, double d, double f){
-				
-		rightLead.config_kP(0, kPRight, TALON_TIMEOUT);
-		leftLead.config_kP(0, kPLeft, TALON_TIMEOUT);
-		
-		rightLead.config_kI(0, kIRight, TALON_TIMEOUT);
-		leftLead.config_kI(0, kILeft, TALON_TIMEOUT);
-		
-		rightLead.config_kD(0, kDRight, TALON_TIMEOUT);
-		leftLead.config_kD(0, kDLeft, TALON_TIMEOUT);
-		
-		LOGGER.info("Right p value: " + kPRight + " and left p value: " + kPLeft);
+		rightLead.setSelectedSensorPosition(0, 0, RobotMap.TALON_TIMEOUT);
+		leftLead.setSelectedSensorPosition(0, 0, RobotMap.TALON_TIMEOUT);
+		
+//		double kPRight = Double.parseDouble(SmartDashboard.getString("DB/String 7", "0"));
+//		double kPLeft = Double.parseDouble(SmartDashboard.getString("DB/String 2", "0"));
+//		
+//		double kIRight = Double.parseDouble(SmartDashboard.getString("DB/String 8", "0"));
+//		double kILeft = Double.parseDouble(SmartDashboard.getString("DB/String 3", "0"));
+//		
+//		double kDRight = Double.parseDouble(SmartDashboard.getString("DB/String 9", "7"));
+//		double kDLeft = Double.parseDouble(SmartDashboard.getString("DB/String 4", "7"));
+//				
+//		rightLead.config_kP(0, kPRight, RobotMap.TALON_TIMEOUT);
+//		leftLead.config_kP(0, kPLeft, RobotMap.TALON_TIMEOUT);
+//		
+//		rightLead.config_kI(0, kIRight, RobotMap.TALON_TIMEOUT);
+//		leftLead.config_kI(0, kILeft, RobotMap.TALON_TIMEOUT);
+//		
+//		rightLead.config_kD(0, kDRight, RobotMap.TALON_TIMEOUT);
+//		leftLead.config_kD(0, kDLeft, RobotMap.TALON_TIMEOUT);
+		
+//			LOGGER.info("Right p value: " + kPRight + " and left p value: " + kPLeft);
 		return false;
 	}
 	
 	public void initMotionMagicMode() {
-		rightLead.setSelectedSensorPosition(0, 0, TALON_TIMEOUT);
-		leftLead.setSelectedSensorPosition(0, 0, TALON_TIMEOUT);
+		rightLead.setSelectedSensorPosition(0, 0, RobotMap.TALON_TIMEOUT);
+		leftLead.setSelectedSensorPosition(0, 0, RobotMap.TALON_TIMEOUT);
 		
-		double kPRight = Double.parseDouble(SmartDashboard.getString("DB/String 7", "0"));
-		double kPLeft = Double.parseDouble(SmartDashboard.getString("DB/String 2", "0"));
+		double kPRight = 1.4; // Double.parseDouble(SmartDashboard.getString("DB/String 7", "1.4"));
+		double kPLeft = 1.6; //Double.parseDouble(SmartDashboard.getString("DB/String 2", "1.6"));
 		
-		double kIRight = Double.parseDouble(SmartDashboard.getString("DB/String 8", "0"));
-		double kILeft = Double.parseDouble(SmartDashboard.getString("DB/String 3", "0"));
+		double kIRight = 0.0; // Double.parseDouble(SmartDashboard.getString("DB/String 8", "0.0"));
+		double kILeft = 0.0; //Double.parseDouble(SmartDashboard.getString("DB/String 3", "0.0"));
 		
-		double kDRight = Double.parseDouble(SmartDashboard.getString("DB/String 9", "7"));
-		double kDLeft = Double.parseDouble(SmartDashboard.getString("DB/String 4", "7"));
+		double kDRight = 165; //Double.parseDouble(SmartDashboard.getString("DB/String 9", "165"));
+		double kDLeft = 198; //Double.parseDouble(SmartDashboard.getString("DB/String 4", "198"));
```

Comment:
Get the default values from RobotMap. Pretty soon you're going to have more than one robot to configure and you'll need these to vary depending on which robot you are running on.